src/app/service/product.service.ts

```diff
@@ -11,6 +11,10 @@ export class ProductService {
     return this.list.filter((item: Product) => item.featured == true).sort(() => 0.5 - Math.random()).slice(0, 5);
   }
 
+  getHomeDiscountsProducts(): Product[] {
+    return this.list.sort(() => 0.5 - Math.random()).slice(0, 5);
```

Comment:
Again, 0.5 - Math.random() could be replaced with something more obvious.
If you need this many times, could be put into a utility function or a method.